requirements/base.txt

```diff
@@ -10,3 +10,6 @@ raven==5.2.0
 requests==2.19.1
 uwsgi==2.0.17.1
 xlrd==0.9.3
+
+#Irat healthcheck and ping package
```

Comment:
_nitpick_ Do we need to mention Irat explicitly? It feels we use the healthcheck/ping for our own purposes, not "Incidence Response and Tuning".

Reply:
Nope. Removed.